packages/grid/docker-compose.multinode.yml

```diff
@@ -0,0 +1,460 @@
+version: "3.8"
+services:
+  # docker-host:
```

Comment:
we might get rid of this file entirely eventually, I used it as a way to start testing the multinode setup